qcodes/station.py

```diff
@@ -132,12 +140,19 @@ def __init__(self, *components: Metadatable,
             self.add_component(item, update_snapshot=update_snapshot)
 
         self.use_monitor = use_monitor
-        self.config_file = config_file
 
         self._added_methods: List[str] = []
         self._monitor_parameters: List[Parameter] = []
 
-        self.load_config_file(self.config_file)
+        if config_file is None:
+            self.config_file = []
+        elif isinstance(config_file, str):
+            self.config_file = [config_file, ]
+        else:
+            assert isinstance(config_file, List)
+            self.config_file = config_file
```

Comment:
```suggestion
            self.config_file = list(config_file)
```
Otherwise this will fail for a tuple and other sequences